bin/cask

```diff
@@ -38,12 +38,8 @@ case $subcommand in
 		;;
 	*)
 		WHICH=${WHICH:-which}
-		READLINK=$($WHICH readlink || true)
-		if [ "$(uname || true)" == "Darwin" ] ; then
-			READLINK=
-		fi
-		READLINK=${READLINK:-$($WHICH greadlink || true)}
-		if [ ! -z "$READLINK" ] ; then
+		READLINK=$($WHICH readlink || $WHICH greadlink)
+		if [ -n "$READLINK" ] && eval "$READLINK" -f "$CASK" &>/dev/null; then
```

Comment:
On an old apple, it'll presumably try `readlink -f`, fail, then python it up.  The old code would have tried `greadlink`.  Ideally, we would try `readlink -f`, fail, try `greadlink`, fail, then python it up.

Maybe `READLINK=$($WHICH greadlink || $WHICH readlink)` would be cleanest.

Reply:
What do you think of using [this](https://stackoverflow.com/a/18443300/226114) to remove python completely?